src/network_service/sending_targets_cache.rs

```diff
@@ -0,0 +1,99 @@
+// Copyright 2019 MaidSafe.net limited.
+//
+// This SAFE Network Software is licensed to you under The General Public License (GPL), version 3.
+// Unless required by applicable law or agreed to in writing, the SAFE Network Software distributed
+// under the GPL Licence is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied. Please review the Licences for the specific language governing
+// permissions and limitations relating to use of the SAFE Network Software.
+
+use crate::ConnectionInfo;
+use std::collections::HashMap;
+use std::net::SocketAddr;
+
+const MAX_RESENDS: u8 = 1;
```

Comment:
It is an arbitrary value, but why 1? as oppose to 3 for example.

Reply:
No reason at all. Just a placeholder value for now, really. 3 is also fine, I'll change it.